lib/api_browser/app/js/factories/Documentation.js

```diff
@@ -1,13 +1,55 @@
-﻿app.factory('Documentation', function($http) {
+/**
+ * This service gives access to the documentation metadata
+ */
+app.factory('Documentation', function($http, $q) {
+  var versions = $q.when(/*REPLACE-WITH-JSON*/$http.get('api/index-new.json', { cache: true }).then(function(data) {
```

Comment:
This should get deleted. It's a concept from the previous PR where the docs index would get bundled and compressed into the javascript.